app/controllers/candidate_interface/after_sign_in_controller.rb

```diff
@@ -29,15 +30,27 @@ def redirect_to_path_if_path_params_are_present
       redirect_to params[:path] if params[:path].present?
     end
 
+    def redirect_to_prefill_if_sandbox_user_has_blank_application
+      if HostingEnvironment.sandbox_mode? && current_application.blank_application?
+        course = course_from_find
```

Comment:
nit: maybe we could lose this — memoize `course_from_find` and consistently refer to it as `course_from_find`?